Dockerfile

```diff
@@ -22,10 +22,10 @@ RUN mkdir /build
 ARG BBLFSHD_VERSION=dev
 ARG BBLFSHD_BUILD=unknown
 
-ENV GO_LDFLAGS="-X main.version=${BBLFSHD_VERSION} -X main.build=${BBLFSHD_BUILD}"
+ENV GO_LDFLAGS="-X main.version=${BBLFSHD_VERSION} -X main.build='${BBLFSHD_BUILD}'"
 
-RUN go build  -tags ostree --ldflags "${GO_LDFLAGS}" -o /build/bblfshd ./cmd/bblfshd/
-RUN go build --ldflags "${GO_LDFLAGS}" -o /build/bblfshctl ./cmd/bblfshctl/
+RUN go build  -tags ostree --ldflags "'${GO_LDFLAGS}'" -o /build/bblfshd ./cmd/bblfshd/
```

Comment:
I figured that having only ` --ldflags '${GO_LDFLAGS}'` does not solve the issue, right?

Reply:
Right—the shell won't interpolate into single quotes. By double-quoting the whole word, the single quotes inside lose their meaning until the substitution is complete, and thus survive to protect it from the subsequent evaluation. 

Shell languages are bad and we rely on them entirely too much. :(